apps/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+
+nodejs_binary(
+    name = "serve",
+    chdir = package_name(),
+    data = [
+        # primary firebase configuration file
+        "firebase.json",
+
+        # Firebase function files
+        "//apps/functions:functions_compiled",
+        "//apps/functions:functions_files",
+    ],
+    entry_point = "@npm//:node_modules/firebase-tools/lib/bin/firebase.js",
+    templated_args = [
+        "--project",
+        "angular-caretaker",
+        "--config",
+        "$$(rlocation $(execpath :firebase.json))",
+        "emulators:start",
+    ],
```

Comment:
That's great! do we know if this will work on Windows? Not that is a strict requirement here, but that would certainly be nice I think.

Reply:
I guess it works since you used `copy_to_bin`? Nice --  I think we should try avoid using this unless really needed (refactoring some FW tests for this -- [see details here](https://angular-team.slack.com/archives/C02PARQNMC1/p1648137933069659)). I think in this case there might be no way around it though, but a comment might be good to mention e.g. Windows?